### ISSUE TITLE
Training CSI using SGD optimizer

### ISSUE THREAD

**Issue by deu30303** [alinlab/CSI#19]
I try to reproduce CSI result in your paper. In my server environment,  LARS does not work. Could you let me know setting using SGD (ex, learning rate, command line etc)?

**Comment by Jihoon-Tack** [comment]
Hi, thank you for your interest!

The settings for SGD and LARS are the same. You can change the `--optimizer lars` to `--optimizer sgd` and run the code.

Thank you

**[update]**
We removed the adam optimizer option since we didn't fully search the training configuration for the adam optimizer.
